### PR TITLE
Adding managed-cluster-config services

### PR DESCRIPTION
**Pull request by rafael-azevedo**
This pr adds mangeed-cluster-config services. Made a new PR since issues with conflict on previous PR https://github.com/app-sre/saas-hive/pull/26.

### CODE REVIEW

File: managed-cluster-config-services/managed-cluser-config.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+services:
+- hash: none
+  name: managed-cluster-config
+  path: /00-osd-manged-cluster-config.selectorsyncset.yaml
```
Comment (jmelis) [inline managed-cluster-config-services/managed-cluser-config.yaml]: `manged` typo?

and also I don't seem to see that file in the repo?

**Review by jmelis — DISMISSED** [review]

File: managed-cluster-config-services/managed-cluser-config.yaml (hunk 1 above)
Comment (rafael-azevedo) [reply managed-cluster-config-services/managed-cluser-config.yaml]: It is a typo. The file is generated by the make command

Comment (jmelis) [reply managed-cluster-config-services/managed-cluser-config.yaml]: unfortunately that doesn't work, it has to be precommited

cc @jharrington22

Comment (rafael-azevedo) [reply managed-cluster-config-services/managed-cluser-config.yaml]: cc @jewzaam

Comment (rafael-azevedo) [reply managed-cluster-config-services/managed-cluser-config.yaml]: I've updated the file path and committed it to the other repo

**Comment by jewzaam** [comment]
LGTM

**Review by jfchevrette — APPROVED** [review]
LGTM

**Pull request merged**